src/rockfish_mcp/server.py

```diff
@@ -721,7 +721,7 @@ async def main():
 
     # Initialize Rockfish client
     api_key = os.getenv("ROCKFISH_API_KEY")
-    base_url = os.getenv("ROCKFISH_BASE_URL", "https://api.rockfish.ai")
+    api_url = os.getenv("ROCKFISH_API_URL", "https://api.rockfish.ai")
```

Comment:
actually, to keep this backward compatible, how about we try ROCKFISH_BASE_URL if ROCKFISH_API_URL is empty?